src/utils/_list.scss

```diff
@@ -0,0 +1,45 @@
+// --- SUI List --- //
+// @include sui-list;
+// @include sui-list('inline');
+// @include sui-list('grow');
+
+@mixin sui-list(
+  $type: 'block'
+){
+  @include reset-list;
+  display: flex;
+
+  @if $type == 'block' {
+    @include sui-list--block;
+  }
+
+  li {
+    @if $type == 'inline' {
+      @include sui-list--inline;
+    }
+
+    @if $type == 'grow' {
+      @include sui-list--grow;
+    }
+  }
+};
+
+@mixin sui-list--block {
```

Comment:
You could move this definitions before using them as a good practice for making it easier to read